src/NumericalAlgorithms/Interpolation/InterpolationTargetApparentHorizon.hpp

```diff
@@ -141,19 +141,30 @@ template <typename InterpolationTargetTag, typename Frame>
 struct ApparentHorizon {
   using const_global_cache_tags =
       tmpl::list<Tags::ApparentHorizon<InterpolationTargetTag, Frame>>;
-  using initialization_tags =
-      tmpl::append<StrahlkorperTags::items_tags<Frame>,
-                   tmpl::list<::ah::Tags::FastFlow,
-                              logging::Tags::Verbosity<InterpolationTargetTag>,
-                              ::ah::Tags::PreviousStrahlkorper<Frame>>,
-                   StrahlkorperTags::compute_items_tags<Frame>>;
+  using initialization_tags = tmpl::append<
+      StrahlkorperTags::items_tags<Frame>,
+      std::conditional_t<
+          std::is_same_v<Frame, ::Frame::Inertial>,
+          tmpl::list<::ah::Tags::FastFlow,
+                     logging::Tags::Verbosity<InterpolationTargetTag>,
+                     ::ah::Tags::PreviousStrahlkorper<Frame>>,
+          tmpl::list<::ah::Tags::FastFlow,
+                     logging::Tags::Verbosity<InterpolationTargetTag>,
+                     ::ah::Tags::PreviousStrahlkorper<Frame>,
+                     StrahlkorperTags::Strahlkorper<::Frame::Inertial>>>,
+      StrahlkorperTags::compute_items_tags<Frame>>;
   using is_sequential = std::true_type;
   using frame = Frame;
 
-  using simple_tags =
+  using simple_tags = std::conditional_t<
+      std::is_same_v<Frame, ::Frame::Inertial>,
       tmpl::push_back<StrahlkorperTags::items_tags<Frame>, ::ah::Tags::FastFlow,
                       logging::Tags::Verbosity<InterpolationTargetTag>,
-                      ::ah::Tags::PreviousStrahlkorper<Frame>>;
+                      ::ah::Tags::PreviousStrahlkorper<Frame>>,
```

Comment:
how about making a type alias `common_tags` so they don't get out of sync in the future.   Then this can be `using simple_tags = tmpl::append<common_tags, tmpl::conditional_t<...>>;`